qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -97,14 +118,49 @@ def __init__(
 
             # Add the basis gates
             for gate in library.basis_gates:
-                self.add_schedule(library[gate])
+                self.add_schedule(library[gate], n_qubits=library.n_qubits(gate))
 
             # Add the default values
             for param_conf in library.default_values():
                 schedule_name = param_conf[-1]
                 if schedule_name in library.basis_gates:
                     self.add_parameter_value(*param_conf)
 
+        self._update_inst_map = True
+
+        # Push the schedules to the instruction schedule map.
+        self.update_inst_map()
+
+    @property
+    def default_inst_map(self) -> InstructionScheduleMap:
+        """Return the default and up to date instruction schedule map."""
+        return self._inst_map
+
+    def get_instruction_schedule_map(
```

Comment:
```suggestion
    def get_inst_map(
```
to fit in with another one?